ui/component/claimPreview/view.jsx

```diff
@@ -504,11 +504,14 @@ const ClaimPreview = forwardRef<any, {}>((props: Props, ref: any) => {
                         <ButtonAddToQueue focusable={false} uri={repostedContentUri} />
                       </div>
                     )}
-                    {(!isLivestream || isLivestreamActive) && (
-                      <div className="claim-preview__file-property-overlay">
-                        <PreviewOverlayProperties uri={uri} small={type === 'small'} xsmall={smallThumbnail} />
-                      </div>
-                    )}
+                    <div className="claim-preview__file-property-overlay">
+                      <PreviewOverlayProperties
+                        uri={uri}
+                        small={type === 'small'}
+                        xsmall={smallThumbnail}
+                        isLivestream={isLivestream}
```

Comment:
The `isLivestream` external prop can/should be avoided -- can get from Redux like all the other properties. This keeps the behavior consistent and self-contained for other instances like the one in `featuredSection` and future ones.